docs/en/connector-v2/sink/Paimon.md

```diff
@@ -41,9 +41,11 @@ libfb303-xxx.jar
 | hdfs_site_path              | String | No       | -                            | The path of hdfs-site.xml                                                                                                                                        |
 | schema_save_mode            | Enum   | No       | CREATE_SCHEMA_WHEN_NOT_EXIST | The schema save mode                                                                                                                                             |
 | data_save_mode              | Enum   | No       | APPEND_DATA                  | The data save mode                                                                                                                                               |
+| changelog-producer          | Enum   | No       | none                         | The changelog producer                                                                                                                                           |
```

Comment:
This duplicates the paimon.table.write-props definition, which can be configured directly in paimon.table.write-props.